doc/tutorials/tutorial_understanding_building_benchmark.rst

```diff
@@ -0,0 +1,312 @@
+.. _Benchopt: https://benchopt.github.io/#
+
+Understanding and building a simple benchmark with Benchopt
+===========================================================
+
+Are you looking for a simple, step-by-step introduction to the `Benchopt`_ library? Then look no further! In this tutorial, we are going to learn how to use the `Benchopt`_ library to compare various algorithms in order to minimize a given cost. This tutorial complements the `write a benchmark <https://benchopt.github.io/how.html>`_ webpage with a more practical, hands-on approach. 
+
+Let's say we want to minimize the cost function of the `ridge regression problem <https://en.wikipedia.org/wiki/Ridge_regression>`_, a variant of linear regression which is a common of supervised machine learning.
```

Comment:
Maybe: Lets' say you're interested in benchmarking solvers to the...

Reply:
typo "which is a common of supervised learning"

Reply:
tried something!